src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/StatsHelper.groovy

```diff
@@ -57,17 +47,78 @@ class StatsHelper {
     }
 
     void verifyYFlowWritesMeterStats(YFlow yFlow, Date from, boolean expectTraffic) {
-        Wrappers.wait(STATS_INTERVAL) {
+        "force kilda to collect stats"(yFlow.getYFlowId())
+        Wrappers.wait(STATS_LOGGING_TIMEOUT) {
             def yFlowId = yFlow.YFlowId
             def dpsShared = otsdb.query(from, metricPrefix + "yFlow.meter.shared.bytes", ["y_flow_id": yFlowId]).dps
             def dpsYPoint = otsdb.query(from, metricPrefix + "yFlow.meter.yPoint.bytes", ["y_flow_id": yFlowId]).dps
+            def dpsSubFlows = otsdb.query(from,
+                    metricPrefix + "flow.raw.bytes",
+                    ["flowid": "${yFlow.subFlows[0].flowId}|${yFlow.subFlows[1].flowId}"]).dps
             if (expectTraffic) {
                 assert dpsShared.values().any { it > 0 }, yFlowId
                 assert dpsYPoint.values().any { it > 0 }, yFlowId
+                assert dpsSubFlows.values().any { it > 2 }, yFlowId
```

Comment:
Method name says verifyYFlowWrites**Meter**Stats but you added rule metric `flow.raw.bytes`
Looks strange

Reply:
Seems like we've found a volunteer to explain rules, meters, metrics and statistics to the team.